_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,238 @@
+---
+title: 'Bitcoin Optech Newsletter #120'
+permalink: /en/newsletters/2020/10/21/
+name: 2020-10-21-newsletter
+slug: 2020-10-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter provides an overview of the new MuSig2 paper,
+summarizes additional discussion about upfront fees in LN, and describes
+a proposal to simplify management of LN payments.  Also included are our
+regular sections with summaries of notable improvements to clients and
+services, announcements of releases and release candidates, and changes
+to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **MuSig2 paper published:** Jonas Nick, Tim Ruffing, and Yannick Seurin
+  published [the MuSig2 paper][musig 2 paper] describing a new variant of the
+  [MuSig][topic musig] signature scheme with a two round signing protocol.
+
+    MuSig is a signature scheme that allows multiple signers to create an
+    aggregate public key derived from their individual private keys, and then
+    collaborate to create a single valid signature for that public key. The
+    aggregate public key and signature are indistinguishable from any other
+    [schnorr][topic schnorr signatures] public key and signature.
+
+    The original version of MuSig required a three round signing protocol:
+    first, the co-signers exchange commitments to nonce values, then they
+    exchange the nonce values themselves, and finally they exchange the
+    partial signatures. This three round protocol is to avoid nonce reuse and
+    related attacks, which could allow a dishonest signer to discover the private
+    key of one of their co-signers. Using deterministic nonces, which is very
+    common practice in single-signer schemes, is unsafe for multi-signer schemes,
+    as described in the [original MuSig blog post][musig blog post unsafe
+    deterministic nonce].  Precomputing the nonces in advance and exchanging them
+    at key setup time is also unsafe, as described in a [blog post by Jonas
```

Comment:
nit: unsafe used in previous sentence
```suggestion
    at key setup time is also dangerous, as described in a [blog post by Jonas
```

Reply:
repeating `unsafe` hear seems fine to me, since it's essentially a technical term.